dts/bindings/lora/st,stm32wl-subghz-radio.yaml

```diff
@@ -12,3 +12,33 @@ properties:
     required: true
     description: |
       Position of the "Radio IRQ, Busy" interrupt line.
+
+  power-amplifier-output:
+    type: string
+    required: true
+    description: |
+      Selects between the low- and high-power power amplifier output pin.
+    enum:
+      - "rfo-lp"
+      - "rfo-hp"
+
+  rfo-lp-max-power:
+    type: int
+    default: 15
+    description: |
+      Maximum design power for the board's RFO_LP output matching network.
```

Comment:
You might want to mention AN5457, so that users know from where those values come from.

Reply:
Good idea, I'll amend the commit.

Reply:
Thanks!